omod/src/main/webapp/pages/visit/templates/overview.gsp

```diff
@@ -1,35 +1,45 @@
 
 <div ng-hide="visit"><i class="icon-spinner icon-spin icon-2x"></i></div>
 
-<span ng-show="visit">
+<div ng-show="visit">
 
-    <div id="visit-details">
-        <visit-details visit="visit"></visit-details>
-    </div>
+    <div class="row justify-content-between">
 
-    <div id="print-actions">
-        <button id="print-button" ng-click="print()" ng-disabled="printButtonDisabled" title="${ ui.message('pihcore.visitNote.print') }"><i class="icon-print"/></button>
-    </div>
+        <div id="visit-details" class="col-6">
+            <visit-details visit="visit"></visit-details>
+        </div>
 
-    <div id="expand-all" ng-show="!allExpanded">
-        <button id="expand-all-button" ng-click="expandAll()" title="${ ui.message('pihcore.visitNote.expandAll') }"><i class="icon-double-angle-down"/></button>
-    </div>
+        <div id="visit-action-buttons" class="col-6 text-right">
 
-    <div id="contract-all" ng-show="allExpanded">
-        <button id="contract-all-button" ng-click="contractAll()" title="${ ui.message('pihcore.visitNote.collapseAll') }"><i class="icon-double-angle-up"/></button>
-    </div>
+            <span id="contract-all" ng-show="allExpanded">
+                <button id="contract-all-button" ng-click="contractAll()" title="${ ui.message('pihcore.visitNote.collapseAll') }"><i class="icon-double-angle-up"/></button>
+            </span>
 
-    <div id="visit-actions">
-        <visit-actions-dropdown></visit-actions-dropdown>
-    </div>
+            <span id="choose-another-visit">
+                <visit-list-dropdown></visit-list-dropdown>
+            </span>
 
-    <div id="choose-another-visit">
-        <visit-list-dropdown></visit-list-dropdown>
-    </div>
+            <span id="visit-actions">
+                <visit-actions-dropdown></visit-actions-dropdown>
+            </span>
+
+            <span id="expand-all" ng-show="!allExpanded">
+                <button id="expand-all-button" ng-click="expandAll()" title="${ ui.message('pihcore.visitNote.expandAll') }"><i class="icon-double-angle-down"/></button>
+            </span>
+
+            <span id="print-actions">
+                <button id="print-button" ng-click="print()" ng-disabled="printButtonDisabled" title="${ ui.message('pihcore.visitNote.print') }"><i class="icon-print"/></button>
+            </span>
+
+        </div>
+
+        <div ng-repeat="encounter in visit.encounters | filter:{voided:false}" class="col-12">
```

Comment:
I think you'll want another div to act as a containing row for this. Right now you've got `row > col > col`

Reply:
Good catch.